Lib/bdb.py

```diff
@@ -36,12 +50,40 @@ def canonic(self, filename):
         return canonic
 
     def reset(self):
+        """Set values of attributes as ready to start debugging."""
         import linecache
         linecache.checkcache()
         self.botframe = None
         self._set_stopinfo(None, None)
 
     def trace_dispatch(self, frame, event, arg):
+        """Trace function of debugged frames.
```

Comment:
Made a slight change to yours to make it a verb, but thank you for suggesting to put 'dispatch' first.  I had struggled with this yesterday.